rewrite/rewrite/python/format/auto_format.py

```diff
@@ -6,7 +6,9 @@
 from .normalize_tabs_or_spaces import NormalizeTabsOrSpacesVisitor
 from .remove_trailing_whitespace_visitor import RemoveTrailingWhitespaceVisitor
 from .spaces_visitor import SpacesVisitor
-from .. import TabsAndIndentsStyle, GeneralFormatStyle
+from .tabs_and_indents_visitor import TabsAndIndentsVisitor
+from .wrapping_and_brances_visitor import WrappingAndBracesVisitor
```

Comment:
It looks like this module is missing from your PR. Also note the typo:
```suggestion
from .wrapping_and_braces_visitor import WrappingAndBracesVisitor
```